pkg/cmd/task/task.go

```diff
@@ -32,9 +40,53 @@ func NewCommand(ctx api.Context) *cobra.Command {
 	cmd.Flags().Bool("completed", false, "Print completed tasks")
 
 	cmd.AddCommand(
+		newCmdTaskAttach(ctx),
 		newCmdTaskList(ctx),
 		newCmdTaskLog(ctx),
 		newCmdTaskLs(ctx),
 	)
 	return cmd
 }
+
+func findTask(ctx api.Context, id string) (*mesosgo.Task, error) {
+	mesosClient, err := mesos.NewClientWithContext(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	allTasks, err := mesosClient.Tasks()
+	if err != nil {
+		return nil, err
+	}
+
+	var tasks []mesosgo.Task
+	for _, t := range allTasks {
+		if strings.Contains(t.TaskID.Value, id) {
```

Comment:
This doesn't go along the design doc: "It then loops through the tasks and tries to find a match which is exactly `<task-id>`, contains`<task-id>`, or matches the wildcard pattern of `<task-id>`."